### PR TITLE
Feature: adds a responsive navigation bar

### PR DESCRIPTION
**Pull request by ixahmedxi**
This commit brings the following changes:

- Replaces the old navigation bar with a new one that is responsive
on mobile devices.

- Moves the MajorLeagueHackingBadge into the navigation bar component,
that way it stays relative to the navigation bar and does not overlap
the menu.

- Removes the yarn.lock file and defaults into using npm throughout the
project.

Changes to be committed:
	modified:   package-lock.json
	modified:   src/components/Header/index.js
	modified:   src/components/Header/index.module.scss
	modified:   src/components/Layout/index.js
	modified:   src/components/MajorLeagueHackingBadge/index.module.scss
	deleted:    yarn.lock

### CODE REVIEW

**Comment by 7coil** [comment]
I'm not sure if we're going to be removing yarn, I'll take a look while I gather people's opinions

**Comment by ixahmedxi** [comment]
The PR can still be merged and if a switch to yarn happens in the future than that can be done as another commit. The current version of having both a yarn.lock and a package-lock.json files is not a good option anyways.

**Comment by 7coil** [comment]
According to a straw poll, 6/6 people want npm over yarn.
Although removing yarn.lock should be done, it would be better if this change was made in a separate pull request, not only to make merges more atomic (which we all love as part of Software Development), but also ~~to get that great Hacktoberfest t-shirt~~.

To do this, you can just temporarily restore yarn.lock and remove package-lock.json in an additional commit, and I'll perform a squash merge.

Then, in a separate merge request, you can delete yarn.lock and add in the package-lock.json

**Comment by ixahmedxi** [comment]
Done

File: src/components/MajorLeagueHackingBadge/index.module.scss [hunk 1]
```diff
@@ -1,3 +1,19 @@
+$breakpoint-md: 768px;
+$breakpoint-lg: 992px;
```
Comment (7coil) [inline src/components/MajorLeagueHackingBadge/index.module.scss:2]: and also same here.

**Review by 7coil — CHANGES_REQUESTED** [review]
Use the Bootstrap SASS mixins please.

**Comment by ixahmedxi** [comment]
I have changed the responsive media queries to bootstrap mixins, I have however chosen to not import the intire bootstrap scss and only include the parts required to make it work that way we don't import unneccessary stuff.

**Review by 7coil — APPROVED** [review]
Everything looks fine now, thanks a lot!

**Pull request merged**